web/index.ts

```diff
@@ -127,7 +127,7 @@ createConnection().then(() => {
         certificateRouter.get("/create/:student/:pupil", authCheckFactory(), certificateController.createCertificateEndpoint);
         certificateRouter.get("/:certificateId", authCheckFactory(), certificateController.getCertificateEndpoint);
         certificateRouter.get("/:certificateId/confirmation", /* NO AUTH REQUIRED */ certificateController.getCertificateConfirmationEndpoint);
-
+        certificateRouter.post("/:certificateId/sign", authCheckFactory(), certificateController.signCertificateEndpoint);
```

Comment:
So with this implementation a pupil (or her parents) need to be authenticated to sign a certificate. Following corona-school/project-user#150 this would mean we need some other kind of authentication included in the link that ensures the user is not forced to login again. This would be ok too, but I could image that making this endpoint somehow publicly available or adding another authentication mechanism here might be simpler. But we can also go with including the _current_ authtoken in the email for now.

Reply:
Ah, I actually missed that requirement. I guess we have similar requirements at other places, so including the current auth token would be a "general solution". The /login frontend already takes the fitting query parameters, right?

Reply:
The certificate (in german) now gets attached to the Mail itself, also the link attached contains the login token, and then opens the settings page (in theory, cannot test the mails :)) so that other languages can be downloaded.